pycomfoconnect/comfoconnect.py

```diff
@@ -212,19 +250,29 @@ def _command(self, command, params=None, use_queue=True):
         self._reference += 1
 
         # Send the message
-        self._bridge.write_message(message)
+        try: 
+            reply = self._bridge.write_message(message)
+        
+        except OSError:
+            _LOGGER.info("Unexpected error in _command._bridge.write_message: ", sys.exc_info()[0])
+            if self._debug: _LOGGER.info(reply)
```

Comment:
You can now get rid of the ._debug, and just always _LOGGER.debug() the message.